privilege/privileges/privileges_test.go

```diff
@@ -372,6 +372,35 @@ func TestShowGrants(t *testing.T) {
 	require.Len(t, gs, 3)
 }
 
+// TestErrorMessage checks that the identity in error messages matches the mysql.user table one.
+// MySQL is inconsistent in its error messages, as some match the loginHost and others the
+// identity from mysql.user. In TiDB we now use the identity from mysql.user in error messages
+// for consistency.
+func TestErrorMessage(t *testing.T) {
```

Comment:
Manually testing this with v5.3.0 and master @ 13d0deb7fef7fd7391dce19f320abef4256223bd show the same error messages as in this test. Are you sure this test fails without your fix?

Reply:
I added this for future checking.

The case where it currently fails prior to this is the Column name in `SHOW GRANTS FOR CURRENT_USER()`. But in the tests I didn't find an example where we check column names.